charts/incubator/splunk/README.md

```diff
@@ -0,0 +1,117 @@
+# ${CHARTNAME}
```

Comment:
While you can ignore these .md files (as they are auto generated), it seems you used the template folder.
That folder is not ready yet for use in such way, please copy an existing App as template instead